.env

```diff
@@ -2,11 +2,13 @@
 # SPDX-License-Identifier: GPL-2.0-only
 # Copyright (c) 2022 Intel Corporation.
 
-lkvs_root=$(cd "$(dirname "$0")" && pwd)
+lkvs_root=$(realpath "$(dirname "${BASH_SOURCE[0]}")")
 
 if [[ -z "$path_exported" ]]; then
-  NEW_PATH="${PATH}":$(find "$lkvs_root" -type d -not -path '*.git*' | tr "\n" ":")
+  NEW_PATH="${PATH}":${lkvs_root}:$(find "$lkvs_root" -type d -not -path '*.git*' | tr "\n" ":")
```

Comment:
Should remove ":${lkvs_root}",  it will show twice in NEW_PATH

Reply:
"${PATH}"   ->   "$PATH",   no need {}  if you already use "".
So change it to as below:
NEW_PATH="$PATH":$(find "$lkvs_root" -type d -not -path '*.git*' | tr "\n" ":")

Reply:
> Should remove ":${lkvs_root}", it will show twice in NEW_PATH

Will remove it.

Reply:
> "${PATH}" -> "$PATH", no need {} if you already use "". So change it to as below: NEW_PATH="$PATH":$(find "$lkvs_root" -type d -not -path '_.git_' | tr "\n" ":")

1. Will remove the braces around **PATH**.
2. Why replace asterisks with underlines around .git?

Reply:
For point2,
Sorry, should change as below:
NEW_PATH="$PATH":$(find "$lkvs_root" -type d -not -path '*.git*' | tr "\n" ":")
Thanks!

Reply:
Ah,  it's git hub's issue,  it will remove * automatically.

`NEW_PATH="$PATH":$(find "$lkvs_root" -type d -not -path '*.git*' | tr "\n" ":")`

Add code sample will avoid this issue.
Thanks!